index.bs

```diff
@@ -948,6 +950,10 @@ these steps.
    requested but not satisfied, then the value is `"default"`
    The getter steps are to return [=this=]'s {{[[CongestionControl]]}}.
 
+: <dfn for="WebTransport" attribute>supportsReliableOnly</dfn>
+:: Returns true if the user agent supports [=WebTransport sessions=] over exclusively reliable [=connections=].
+   [=connections=], otherwise false.
```

Comment:
```suggestion
:: Returns true if the user agent supports [=WebTransport sessions=] over exclusively reliable
   [=connections=], otherwise false.
```